README.md

```diff
@@ -34,7 +35,7 @@ Building the code
 2) Check out a copy of the source:
 
    ```
-   $ git clone git://github.com/H-uru/dirtsand.git dirtsand
+   $ git clone https://github.com/H-uru/dirtsand.git
```

Comment:
Was there a reason you removed the `dirtsand` target creation directory part of the `clone` command? I'd think you'd need this to be the location cloned into, otherwise the next `cd dirtsand` won't be valid.

Reply:
Either way works here. If you don't explicitly give a directory name, Git derives it from the remote URL. In this case the remote repository is called `dirtsand.git`, so the automatic directory name is also `dirtsand`, meaning you can leave off the directory name and save a bit of typing 🙂